tests/www/presenters/data_generators.py

```diff
@@ -44,6 +45,8 @@ def get_plan(
             price_per_unit = Decimal(5)
         if labour_cost_per_unit is None:
             labour_cost_per_unit = Decimal(1)
+        if is_expired is None:
```

Comment:
why are all args Optional and then set if they are None anyway here?

Reply:
You have a point here.  While this procedure is necessary for plan_id or activation_date (because it guarantees that the UUIDs and the datetimes are generated freshly on every method call), it does not make sense for immutable values like bools, strings or Decimals. I change it.